FrontEnd/src/components/profileList/ProfileListPage.module.css

```diff
@@ -0,0 +1,21 @@
+.page {
+  width: 1512px;
```

Comment:
It's very bad to hardcode page width. users with smaller or bigger screens will be affected

Reply:
removed, ty